controls/bsi_app_4_4.yml

```diff
@@ -0,0 +1,355 @@
+policy: 'BSI-APP-4-4'
+title: 'BSI APP.4.4 Kubernetes'
+id: bsi_app_4_4
+version: '1.0'

```

Comment:
Indeed, they are versioned by year. The 2023 edition is the most recent one. I believe the 2022 and 2023 release of APP.4.4 and SYS.1.6 are identical, however. English releases are performed less frequent, as they are not "official documents".

I believe we can use `version: '2023'` here, what do you think @sluetze?

Reply:
you are right, there was no update in the building blocks in the 2023 edition (which is only available in german at the moment). But since we link to the 2022 edition (because its english) I would stay with 2022 and update to 2023 as soon as 2023 is released in english.